requirements.txt

```diff
@@ -1,2 +1,3 @@
 qiskit-nature>=0.6
+qiskit-algorithms>=0.2.1
```

Comment:
```suggestion
```
There is no need for this change since it is required by `qiskit-nature` already.
Thanks, @woodsp-ibm, for pointing this out offline :+1: 

I will apply this change when this PR becomes unblocked.